### PR TITLE
support both, single command or list of commands for apoc.systemdb.execute

### PR DESCRIPTION
**Pull request by sarmbruster**

### CODE REVIEW

File: full/src/main/java/apoc/systemdb/SystemDb.java [hunk 1]
```diff
@@ -63,9 +64,23 @@ public NodesAndRelationshipsResult(List<Node> nodes, List<Relationship> relation
...
+            commands = (List<String>) ddlStringOrList;
+        } else {
+            throw new IllegalArgumentException("don't know how to handle " + ddlStringOrList + ". Supply either a string or a list of strings");
+        }
+
+        Transaction tx = apocConfig.getSystemDb().beginTx();  // we can't use try-with-resources otherwise tx gets closed too early
+        return commands.stream().flatMap(command -> tx.execute(command, params).stream().map(RowResult::new)).onClose(() -> {
+            tx.commit();
+            tx.close();
+        });
```
[the remark this answers is not in the document]
Comment (sarmbruster) [reply full/src/main/java/apoc/systemdb/SystemDb.java:87]: rollback happens automatically if `tx.execute` fails. I've added a check to only do `tx.commit` if the tx is still open, see below.

**Comment by jexp** [comment]
Just curious where did the request come from? 
Compared with executing multiple calls to the procedure?

For the other apoc.cypher.* we don't have list support either?
The only one is run-multi which uses a string but then breaks on `;$`

**Comment by eastlondoner** [comment]
From me - I'd like to be able to run multiple system dB queries in a single transaction in initialiser. I need either all to succeed or none to succeed so the system dB is not left in an unsatisfactory intermediate state

**Comment by eastlondoner** [comment]
Semicolon delimited would be fine

**Comment by conker84** [comment]
@sarmbruster ping

**Pull request merged**